Makefile

```diff
@@ -1,21 +1,28 @@
+BUNDLE_PATH = "caffeine@patapon.info.zip"
+EXTENSION_DIR = "caffeine@patapon.info"
+
 all: build install
 
-.PHONY: build install lint dist
+.PHONY: build install lint lint-fix
 
 build:
-	./update-locale.sh
```

Comment:
Shouldn't we need to keep this ? Not so sure...

Reply:
Ah I guess `gnome-extensions pack` does this!

Reply:
Yep, `gnome-extensions pack` generates the .mo files, and puts them directly into the bundle. This means we don't need the whole `[LANGAUGE_CODE]/LC_MESSAGES/gnome-shell-extension-caffeine.po`, as it generates them straight from `[LANGUAGE_CODE].po`.

Since we don't need to generate the files manually to make the bundle, we don't need to call `./update-locale.sh` every build, so `make build` won't change the translation files, it'll leave the source alone and just build the extension.